linode/manager
# chore: [M3-6772] - Clean up one-off link styles

## Description 📝
The majority of this ticket was taken care of in #9411 to apply the correct color to the Link component, freeing us from the need to wrap the link in `Typography` to inherit styles or add custom styling for the right color. Now that the Link component has the correct colors defined, this PR cleans up a few instances of `color: theme.textColors.linkActiveLight` styling where it is no longer needed.

## Preview 📷

No change:

| Prod  | This Branch   |
| ------- | ------- |
| ![Screenshot 2023-08-09 at 12 43 59 PM](https://github.com/linode/manager/assets/114685994/8a15db04-4bcd-4ee0-a516-913106413a0a) | ![Screenshot 2023-08-09 at 12 43 49 PM](https://github.com/linode/manager/assets/114685994/f884f8e1-38ea-4202-8c36-b2161dce485c) |
| ![Screenshot 2023-08-09 at 12 44 07 PM](https://github.com/linode/manager/assets/114685994/3eb23be9-8282-47f2-bea4-9b3c7af1b289) | ![Screenshot 2023-08-09 at 12 44 15 PM](https://github.com/linode/manager/assets/114685994/a618ff9f-a39d-4620-9b98-1633ac7a1afb) |

## How to test 🧪
1. **How to verify changes?**
- Confirm that there have been no visual regressions in Link colors with the above changes in _either light or dark mode_:
   - IconTextLink: http://localhost:3000/longview/clients/{id}
   - FirewallRow: http://localhost:3000/firewalls
   - KubernetesClusterRow: http://localhost:3000/kubernetes/clusters/{id}/summary
   - LinodeEntityDetail: http://localhost:3000/linodes/{id}
   - Linode Row: http://localhost:3000/linodes
   - BucketTableRow: http://localhost:3000/object-storage/buckets
   - ResultRow: search bar
   - ToastNotifications: this one's kind of a pain to test, so see the Preview screenshots
